modules/ListView.js

```diff
@@ -22,7 +21,44 @@
    */
   var ListView = View.extend((function() {
 
-    var removeChildView, addChildView;
+    var removeChildView, addChildView, aggregateRenders, breakDelayedRender;
+
+    /**
+     * If one exists, this method will clear the delayed render timeout and invoke render 
+     * @param view {List View} the list view
+     * @private
+     * @method breakDelayedRender
+     */
+    breakDelayedRender = function(view) {
+      if (view._delayedRenderTimeout) {
+        clearTimeout(view._delayedRenderTimeout);
+        view._delayedRenderTimeout = null;
+        view.render();
+      }
+    };
+
+    /**
+     * Aggregates calls to render by waiting a certain amount of time and then rendering.
+     * Calls that happen while it is waiting, will be swallowed. Useful for when you want to
+     * batch render calls
+     * @private
+     * @method aggregateRenders
+     * @param wait {Numeric} the number of milliseconds to wait before rendering
+     * @param view {List View} the list view
+     */
+    aggregateRenders = function(wait, view) {
+      var postpone = function() {
+        view._delayedRenderTimeout = null;
+        view.render();
+      };
+      return function() {
+        if (!view._delayedRenderTimeout && wait > 0) {
+          view._delayedRenderTimeout = setTimeout(postpone, wait);
+        } else if (wait === 0) {
```

Comment:
yea, just to validate the input before using it.  I could see using wait = -1 for no wait.